redaxo/src/core/tests/packages/manager_test.php

```diff
@@ -18,6 +18,86 @@ public function testMatchVersionConstraints($expected, $version, $constraints)
         static::assertSame($expected, $method->invoke(null, $version, $constraints));
     }
 
+    public function testInstallReturnsFalse()
+    {
+        $addonMock = $this->getMockBuilder(rex_addon::class)
+            ->setConstructorArgs(['Fake'])
+            ->getMock();
+
+        $packageManager = rex_addon_manager::factory($addonMock);
+
+        $result = $packageManager->install();
+        static::assertFalse($result);
+    }
+
+    public function testInstallMessageIsNotEmpty()
```

Comment:
danke für diese tests.. ich denke wir können uns daran orientieren um selbst welche für die installtionsroutine zu bauen.. für das konkrete hier gebaute feature ist vermutlich aber erstmal kein test in dieser form nötig.